corehq/form_processor/models.py

```diff
@@ -199,8 +199,8 @@ class XFormInstanceSQL(DisabledDbMixin, models.Model, RedisLockableMixIn, Attach
     # The time at which the server has received the form
     received_on = models.DateTimeField()
 
-    auth_context = JSONField(lazy=True, default=dict)
-    openrosa_headers = JSONField(lazy=True, default=dict)
+    auth_context = JSONField(default=dict)
+    openrosa_headers = JSONField(default=dict)
```

Comment:
It looks like the new version does lazy loading by default.
Old:
https://github.com/derek-schaefer/django-json-field/blob/master/json_field/fields.py#L118
New:
https://github.com/bradjasper/django-jsonfield/blob/master/jsonfield/fields.py#L69